### ISSUE TITLE
OIP3 - Get rid of useless furniture/stuff

### ISSUE THREAD

**Issue by leonardoalt** [ethdevberlin/OIPS#3]
Mostly the lounge area and the back meeting room have a lot of furniture that just exist and do not have a function. I'd suggest to get rid of those.
Examples:
- Yellow seats that are just folded behind the yellow couch
- Yellow couch standing vertically in the meeting room
- Metal cabinet next to the elevator door
- Metal shelves next to the glass door
- White tv bank in the meeting room
- White small table in the meeting room
- Old cabinet next to the meeting room door
- Stool next to the meeting room door
- Stools next to the entrance door
- White carpet "things" all over the lounge
- Lamp between the table and tv

### Motivation
They're not used and take space.

### Backwards Compatibility
Most of the things I listed are not and were never used (at least in this office), so I don't think they will eventually be useful.

If someone argues that these items are "interior design elements" I will disagree but accept.

Though I'm prepared to use the "lean EF" argument :p

**Comment by ligi** [comment]
good point. But let's please not throw things away. Let's put it on ebay-kleinanzeigen or so that others that find it useful can use it. And perhaps with buy some good tea, food or other things with the money we get for these things ;-)

**Comment by Marenz** [comment]
I might be interested in some of the things listed...

**Comment by ligi** [comment]
great! we need to check with @mvglasenapp - will bring it up tomorrow - could not make it to the office today as I had a call and do not like the small conference rooms for these - calls are better on my home setup.

**Comment by Marenz** [comment]
There are also the folded black/orange chairs next to the elevator door. If we at least move that stuff to the elevator storage room, we can store the drink boxes there

**Comment by mvglasenapp** [comment]
My position here is that I wouldn't get rid of something as long as it doesn't interfere, especially if ever it could be useful one day. Of course, we can discuss this.

Examples:
    - Yellow seats that are just folded behind the yellow couch => was and might be used again
    - Yellow couch standing vertically in the meeting room => reserved by Felix since a few weeks
    - Metal cabinet next to the elevator door => available if no other objection
    - Metal shelves next to the glass door => could be useful one day, doesn't interfere
    - White tv bank in the meeting room => could be useful one day, doesn't interfere
    - White small table in the meeting room => could be useful one day, doesn't interfere
    - Old cabinet next to the meeting room door => could be useful one day, doesn't interfere and "interior design elements" ;)
    - Stool next to the meeting room door => was and might be used again
    - Stools next to the entrance door => was and might be used again
    - White carpet "things" all over the lounge => "interior design elements" but we can discuss this all together ;)
    - Lamp between the table and tv => could be useful one day, doesn't interfere

Btw, interesting initiative just before the crypto WG setup ;)

**Comment by ligi** [comment]
ping @fjl regarding the couch

**Comment by ligi** [comment]
and nice to see we have our first contentious OIP ;-)

**Comment by leonardoalt** [comment]
`especially if ever it could be useful one day`
That's what I meant by "Most of the things I listed are not and were never used (at least in this office)". I don't think keeping them forever just because they might one day be useful is a good thing.

`- Yellow seats that are just folded behind the yellow couch => was and might be used again`
I've never personally seen them being used, but fair enough.

```
- Metal shelves next to the glass door => could be useful one day, doesn't interfere
- White tv bank in the meeting room => could be useful one day, doesn't interfere
- White small table in the meeting room => could be useful one day, doesn't interfere
- Old cabinet next to the meeting room door => could be useful one day, doesn't interfere and "interior design elements" ;)
- Lamp between the table and tv => could be useful one day, doesn't interfere
```
All this stuff together interferes regarding space. Each element alone is small/medium, but together they pollute the general space.

In general, I disagree with the argument `might be useful one day`, as it goes in the direction of hoarding.

`Btw, interesting initiative just before the crypto WG setup ;)`
Haha I personally do not want any of the things I listed.
